### PR TITLE
SWAG all the things

### PR DESCRIPTION
**Pull request by pdyraga**
Turn on the SWAG on Etherscan. Example: https://etherscan.io/address/0x27321f84704a599ab740281e285cc4463d89a3d5#code

image

### CODE REVIEW

File: contracts/governance/Checkpoints.sol [hunk 1]
```diff
@@ -1,4 +1,20 @@
...
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//
+//
+//           Trust math, not hardware.
```
Comment (cygnusv) [inline contracts/governance/Checkpoints.sol:17]: Let's find here a common motto. We can use the Threshold branding material.

Comment (pdyraga) [reply contracts/governance/Checkpoints.sol:17]: Hmmm... I was sure this is what we ended up with. Tagging @dougvk and @willthesis here.

Comment (dougvk) [reply contracts/governance/Checkpoints.sol:17]: @cygnusv the branding material isn't for our code. do you have another suggestion? we can discuss on discord if needed

Comment (pdyraga) [reply contracts/governance/Checkpoints.sol]: Removed the motto, we can always add it later again.

File: contracts/governance/GovernorParameters.sol [hunk 2]
```diff
@@ -1,4 +1,17 @@
-// SPDX-License-Identifier: MIT
+// SPDX-License-Identifier: GPL-3.0-or-later
```
Comment (pdyraga) [inline contracts/governance/GovernorParameters.sol:1]: Flagging this change @cygnusv - did we have MIT there on purpose? I was sure we are going with GPL everywhere?

Comment (cygnusv) [reply contracts/governance/GovernorParameters.sol:1]: That's a mistake 🙈

Comment (pdyraga) [reply contracts/governance/GovernorParameters.sol:1]: So it's fixed now :)

**Review by cygnusv — APPROVED** [review]
SWAG it!

**Pull request merged**